x-pack/plugins/security_solution/public/app/home/home_navigations.ts

```diff
@@ -168,27 +168,24 @@ export const navTabs: SecurityNav = {
     disabled: false,
     urlKey: 'administration',
   },
-  [SecurityPageName.threatIntelligence]: {
-    id: SecurityPageName.threatIntelligence,
-    name: i18n.THREAT_INTELLIGENCE,
-    href: APP_THREAT_INTELLIGENCE_PATH,
-    disabled: false,
-    urlKey: 'threat_intelligence',
+  [SecurityPageName.threatIntelligenceIndicators]: {
```

Comment:
similarly to the CSP plugin below, most of the values are now coming from the Threat Intelligence plugin directly